test/util/preconditions.js

```diff
@@ -52,7 +52,8 @@ describe('preconditions', function() {
       $.checkArgumentType(1, PrivateKey);
     } catch (e) {
       error = e;
-      e.message.should.equal('Invalid Argument for (unknown name), expected PrivateKey but got number');
```

Comment:
The message stays the same for browsers that support `constructor.name`, and likewise a toString() method.